src/core/Table/TableRowMemoized.tsx

```diff
@@ -70,6 +79,38 @@ const TableRow = <T extends Record<string, unknown>>(props: {
 
   const refs = useMergedRefs(rowRef, mergedProps.ref);
 
+  const subRowExpanderCellIndex = row.cells.findIndex(
+    (c) => c.column.id !== SELECTION_CELL_ID,
+  );
+
+  const getSubRowExpander = (cell: Cell<T>) =>
+    expanderCell ? (
+      expanderCell({
+        ...tableInstance,
+        ...{ cell, row: cell.row, value: cell.value, column: cell.column },
+      })
+    ) : (
+      <IconButton
+        style={{ marginLeft: cell.row.depth * 43, marginRight: 16 }}
+        className='iui-row-expander'
+        styleType='borderless'
+        size='small'
+        onClick={(e) => {
```

Comment:
This should be memoized, no?

Reply:
It is small component, not really worth.